src/rard/static/css/project.css

```diff
@@ -348,3 +348,14 @@ button.insert {
   display: flex;
   justify-content: space-between;
 }
+
+#bib-inline-form.htmx-added {
+  opacity: 0;
+  max-height: 0;
+}
+
+#bib-inline-form {
+  opacity: 1;
+  max-height: 1000px;
```

Comment:
Generally speaking it's better to not use px. 50vh might be better if this is semi-arbitrary; it can only take up to half of the screen

Reply:
Ok, I shall